spec/support/migrations_shared_context.rb

```diff
@@ -0,0 +1,5 @@
+shared_context 'migrations' do
+  before do
+    stub_const 'Cassanity::Config::CONFIG_FILE', 'spec/support/cassanity.erb.yml'
```

Comment:
This is why I don't like the hard coded config/config file choice. Having to override a constant feels wrong. Having defaults is good, but I'd prefer more explicit config that gets passed in wherever the tasks are used.